changed common logs logic using regex pattern matching

rimaulana opened the pull request:
I changed the logic for common logic selection to use regex pattern matching, By doing this we will be able to select multiple files that have the same pattern. For example kube-proxy log. As per your, I have tested this modification on:

Kubernetes cluster created by EKS console 
- v1.11 ami-0c5b63ec54dd3fc38 (us-east-1)
- v1.10 ami-0a3df91af7c8225db (ap-southeast-1)

Kubernetes cluster created by eksctl
- v1.11 ami-0b10ebfc82e446296 (us-east-2)
- v1.10 ami-0d885462fa1a40e3a (us-east-2)

rimaulana committed:
changed common logs logic using regex pattern matching

nithu0115 commented:
@rimaulana - would you explain the reason behind regex ? I see this would collect multiple files of same pattern but I am unable understand the rationale behind this.

nithu0115 commented:
@rimaulana - as discussed, please remove regex and have kube-proxy log to common-log.

rimaulana committed:
removed regex pattern and added kube-proxy.log

rimaulana committed:
Merge branch 'fix/regex' into regex

rimaulana commented:
Removed regex pattern and added kube-proxy.log into common-log. I have tested the change on cluster created by console as well as eksctl for both v1.11 and v1.10

nithu0115 commented:
Thanks @rimaulana 👍 

LGTM!
